File: Lab6/t4.java
public class t4{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int i, j, k, num;
        System.out.print("How many lines should we draw? ");
        num = scan.nextInt();
        scan.close();
        for (i = 0; i <= num; i++){
            for (j = 0; j <= num - i; j++)
                System.out.print(" ");
            for (k = 0; k < 2 * i - 1; k++)
                System.out.print("*");
            System.out.println();
        }
    }
}

File: Lab6/t4.cpp
#include <stdio.h>
int main(){
    int i, j, k, num;
    printf("How many lines should we draw? ");
    scanf("%d", &num);
    for (i = 0; i <= num; i++){
        for (j = 0; j <= num - i; j++)
            printf(" ");
        for (k = 0; k < 2 * i - 1; k++)
            printf("*");
        puts("");
    }
    return 0;
}